client/src/app/model/reducer.js

```diff
@@ -1,8 +1,11 @@
 import { combineReducers } from 'redux';
 
-import items, { uiStatus } from 'modules/shopping-list/model/reducer';
+import items, {
```

Comment:
shouldn't we have two separate reducers for items (products) and shopping lists?